### PR TITLE
Bug 1558794, specify image prefix during upgrade

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1558794

### CODE REVIEW

**Comment by ganhuang** [comment]
@ahardin-rh Please hold on to merge into branch 3.10. There were some changes made against 3.10 that should not require those setting any more. Please see https://bugzilla.redhat.com/show_bug.cgi?id=1516534

**Comment by ganhuang** [comment]
LGTM for 3.9.

**Comment by ahardin-rh** [comment]
@ganhuang Okay, thank you!

**Comment by ahardin-rh** [comment]
@openshift/team-documentation Please peer review

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -469,6 +469,20 @@ Automated Upgrade] to ensure you are using the latest upgrade playbooks.
 . Ensure the `openshift_deployment_type` parameter in your inventory file is set
 to `openshift-enterprise`.
 
+. Starting with {product-title} 3.9, the {product-title} web console is deployed
+as a pod on masters during upgrade, and the `openshift_web_console_prefix` is
+introduced to deploy the web console with a customized image prefix. The
+`template_service_broker_prefix` is updated to match other components. If you
+are using a customized *docker-registry* for your installation instead of
```
Comment (kalexand-rh) [inline upgrading/automated_upgrades.adoc]: s/are using/use

File: upgrading/automated_upgrades.adoc [hunk 2]
```diff
@@ -469,6 +469,20 @@ Automated Upgrade] to ensure you are using the latest upgrade playbooks.
...
+*_registry.access.redhat.com_*, you must specify `openshift_web_console_prefix` and
```
Comment (kalexand-rh) [inline upgrading/automated_upgrades.adoc]: I'd say "you must explicitly specify values for `those_parameters`..." but :woman_shrugging:

**Review by kalexand-rh — COMMENTED** [review]
A pick, but otherwise LGTM

**Pull request merged**